### ISSUE TITLE
Resources from an object

### ISSUE THREAD

**Issue by Djspaceg** [trim21/userscript-metadata-generator#4]
I'm using an object to create a collection of resources but the output being generated is incorrect.

Using a string or an array of strings for `@resource` works as expected, producing the `@resource   resourceName. resourceUrl`, but when using an object (keys for names and values for URLs), incorrect output is produced:

### Example

Given the following cases, the subsequent output is returned.

As an array of strings:
```javascript
  resource: [
    'icon-example-1  https://my.website.com/resource1.png',
    'icon-example-2  https://my.website.com/resource2.png',
  ],
```

Correct:
```javascript
// @resource      icon-example-1  https://my.website.com/resource1.png
// @resource      icon-example-2  https://my.website.com/resource2.png
```

As an object
```javascript
  resource: {
    'icon-example-1': 'https://my.website.com/resource1.png',
    'icon-example-2': 'https://my.website.com/resource2.png',
  },
```

Incorrect:
```javascript
// @icon-example-1icon-example-1 https://my.website.com/resource1.png
// @icon-example-2icon-example-2 https://my.website.com/resource2.png
```

The problematic part is this little fella: `@icon-example-1icon-example-1` which appears to be the name key repeated twice instead of `@resource` followed by the name.

Thanks for your time. Hopefully this report helps. Please let me know if you have any questions.

**Comment by trim21** [comment]
are you using latest 0.0.11?

Therre is a test case for this and it looks fine...

**Comment by trim21** [comment]
can I can't reproduce it with current head

```js
  test("resource issue 4", function () {
    expect(
      generateLines({
        resource: {
          'icon-example-1': 'https://my.website.com/resource1.png',
          'icon-example-2': 'https://my.website.com/resource2.png',
        },
      }),
    ).toEqual([
      '// ==UserScript==',
      '// @resource   icon-example-1 https://my.website.com/resource1.png',
      '// @resource   icon-example-2 https://my.website.com/resource2.png',
      '// ==/UserScript==',
    ]);
  })
```

**Comment by trim21** [comment]
Yeah, now I'm pretty sure this has been fixed and released in 0.0.11, please upgrade your version.

**Comment by Djspaceg** [comment]
In case someone else encounters a situation like this in the future, it looks like the issue boiled-down to be a NodeJS version issue. I was previously using v16.x and WebPack 5, like the docs said, but for some reason I couldn't get beyond v0.1.4 of this module until _after_ upgrading to NodeJS v18.x. My original issue is now resolved. Thanks again!
